src/fontra/views/editor/panel-selection-info.js

```diff
@@ -160,6 +164,21 @@ export default class SelectionInfoPanel extends Panel {
 
     const formContents = [];
     if (glyphName) {
+      formContents.push({
+        type: "header",
+        label: "Glyph info",
+        auxiliaryElement: html.createDomElement("icon-button", {
+          "id": "glyphLocking",
+          "style": `width: 1.3em;`,
+          "src":
+            glyphLocked || this.fontController.readOnly
+              ? "/tabler-icons/lock.svg"
+              : "/tabler-icons/lock-open-2.svg",
+          "onclick": (event) => this._toggleGlyphLock(varGlyphController.glyph),
+          "data-tooltip": glyphLocked ? "Unlock glyph" : "Lock glyph",
```

Comment:
The tooltip should read "Glyph is read-only" in the `this.fontController.readOnly` case.

Reply:
done